packages/webpack-cli/lib/utils/get-package-manager.js

```diff
@@ -32,7 +49,16 @@ function getPackageManager() {
         // Nothing
     }
 
-    return 'npm';
+    try {
+        // the sync function below will fail if pnpm is not installed,
+        // an error will be thrown
+        if (sync('pnpm', ['--version'])) {
+            return 'pnpm';
+        }
+    } catch (e) {
+        logger.error('No package manager found.');
+        process.exitCode = 2;
```

Comment:
I think we need run `process.exit(2)` here, we should not continue work

Reply:
I tried that earlier but the unit test seems to fail too early.
![Screenshot at 2020-11-09 18-06-05](https://user-images.githubusercontent.com/46647141/98542041-51668e80-22b6-11eb-8a05-efcaf1d5113b.png)

Reply:
Yes, please mock `process` in this case